runtimes/starlarkrt/runtime/builtins.go

```diff
@@ -20,17 +22,69 @@ var builtins = map[string]func(thread *starlark.Thread, fn *starlark.Builtin, ar
 
 		value, err := starlark.Call(thread, fn, args[1:], kwargs)
 		if err != nil {
+			if perr, ok := sdktypes.FromError(err); ok {
+				v := perr.Value()
+				vctx := values.Context{}
+				if slv, cerr := vctx.ToStarlarkValue(v); cerr == nil {
```

Comment:
if cerr is not nil ?
you want to return just the original err ?
if it's intentional you can resolve this thread

Reply:
if cerr is not nil it means that the value is unconvertable to starlark, so i just string it into the error, no info lost.